### ISSUE TITLE
[vite:css] [less] Unrecognised input 

### ISSUE THREAD

**Issue by Jzow** [vbenjs/vue-vben-admin#3010]
**⚠️ IMPORTANT ⚠️ Please check the following list before proceeding. If you ignore this issue template, your issue will be directly closed.**

- [x] Read [the docs](https://anncwb.github.io/vue-vben-admin-doc/).
- [x] Make sure the code is up to date. (Some bugs have been fixed in the latest code)
- [ ] This is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/anncwb/vue-vben-admin/discussions) or join our [Discord](https://discord.gg/8GuAdwDhj6) Chat Server.

### Describe the bug

A clear and concise description of what the bug is..

### Reproduction

Please describe the steps of the problem in detail to ensure that we can restore the correct problem

## System Info

- Operating System: win x86
- Node version: 18
- Package manager (npm/yarn/pnpm) and version: npnm

```
 ERROR  [vite:css] [less] Unrecognised input                                                                                                                                                    11:21:24  
file: E:/opensource/xxxxx/src/design/index.less:36:22
```

https://github.com/vbenjs/vue-vben-admin/blob/1c668f21bf7ddfbc60c194e3427ea2674aa1ee16/src/design/index.less#L39

**Comment by Jzow** [comment]
pnpm build report error

**Comment by Jzow** [comment]
CC :
1. https://github.com/vitejs/vite/issues/14350

**Comment by Jzow** [comment]
Fied

**Comment by Jzow** [comment]
Fixed, this PR has done two things:

1. We have modified some `less` file naming conventions under the `design module` while also modifying the names referenced by` vue component` files. The purpose of this is that we should not follow the practice of non-standard `vben-ui` naming. We can make appropriate adjustments to facilitate future extensions of new `css` and `less` files that also need to comply with the regulations.
2. Add CI/CD for Github Action.
